pkg/cloud-provider/cloudapi/internal/cloud_common.go

```diff
@@ -228,3 +234,36 @@ func (c *cloudCommon) GetStatus(accountNamespacedName *types.NamespacedName) (*c
 
 	return accCfg.GetStatus(), nil
 }
+
+func (c *cloudCommon) AddInventoryPoller(accountNamespacedName *types.NamespacedName) error {
```

Comment:
Comments on the new functions

Reply:
All new functions have comments, plz check cloud.go . If we follow through the call stack, have comments for all of them.As many of them are just wrappers, we would be repeating the same comment.